Dx 944 cobadge card (#393)
SwedbankPay/developer.swedbankpay.com: payments/card/other-features.md

{% include prices.md %}

## Co-badge Card Choice for Dankort

> Due to EU regulations from 2016-06-09 regarding cards that have more than

Reviewer: Perhaps add the link you found as a reference from here?

```suggestion
Due to [EU regulations from 2016-06-09][eu-regulations] regarding cards that have more than
```

{% include prices.md %}

## Co-badge Card Choice for Dankort

> Due to new EU rules (09.06.2016) regarding cards that have
> more than one payment application, we have developed support for the end users of
> Dankort to be able to choose their preferred payment application on the
> SwedbankPay payment page.
> Read more about this EU legislation at [Dankort][dankort-eu].
> 
> As a merchant you are able to set a priority selection of payment application by
> contacting [SwedbankPay Support][swedbankpay-support]. The end user will always
> be able to override this priority selection on the payment page.
> 
> If you want more information about Co-badge Card Choice for Dankort users please contact
> [SwedankPay Support][swedbankpay-support]. The example below shows the
> payment window where there payer can select between Dankort or Visa before completing
> the payment.

Reviewer: ```suggestion
Due to EU regulations from 2016-06-09 regarding cards that have more than
one payment application, we have developed support for the end users of Dankort
to be able to choose their preferred payment application on the Swedbank Pay
payment page. Read more about this EU legislation at [Dankort][dankort-eu].

As a merchant you are able to set a priority selection of payment application by
contacting [SwedbankPay Support][swedbankpay-support]. The end user will always
be able to override this priority selection on the payment page.

If you want more information about Co-badge Card Choice for Dankort users please
contact [SwedankPay Support][swedbankpay-support]. The example below shows the
payment window where there payer can choose between Dankort or Visa before
completing the payment.
```

> ![card-badge][card-badge]{:height="500px" width="425px"}

Reviewer: ```suggestion
![Co-badge Dankort cards with option to choose between Dankort and Visa before paying][card-badge]{:height="500px" width="425px"}
```

[cancel]: /payments/card/after-payment#cancellations
[capture]: /payments/card/capture
[callback]: /payments/card/other-features#callback
[card-badge]: /assets/img/card-badge.png

> [dankort-eu]:  https://www.dankort.dk/Pages/Dankort-eller-Visa.aspx

Reviewer 2: ```suggestion
[dankort-eu]: https://www.dankort.dk/Pages/Dankort-eller-Visa.aspx
```
